src/app/app.service.ts

```diff
@@ -30,10 +30,10 @@ export class AppService {
         app.use(helmet());
 
         const corsWhitelist = process.env.CORS_WHITELIST;
-        if (corsWhitelist == null || corsWhitelist === '') {
-            app.enableCors();
+        if (corsWhitelist == null || corsWhitelist === null || corsWhitelist === '') {
```

Comment:
I dont think this line is necessary for change (not without an additional test of corsWhiteList === undefined)

Reply:
would
```
corsWhitelist === undefined || corsWhitelist === null || corsWhitelist === ''
```

be clearer?

Reply:
Adding this for the record: It’s not very long, it’s pretty easy to grok the author’s intent either way. The question is more how hard will a casual observer need to think to determine the correctness of the code, and using === seems more explicit.